scripts/pyro_camera_server.py

```diff
@@ -6,10 +6,11 @@
 but before POCS.
 """
 import argparse
-
 from huntsman.utils.pyro import run_camera_server
 
 if __name__ == "__main__":
+        
+    #Run the camera server
```

Comment:
Eventually... the code immediately following this comment is command line argument parsing, though.